src/systemtest/java/util/PropertiesHandler.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.Properties;
+
+public class PropertiesHandler {
+
+    public static void setProperties() throws Throwable {
+        final String systemTestFilePath;
+
+        if (System.getProperty("propertiesFile") != null) {
+            systemTestFilePath = System.getProperty("propertiesFile");
+        } else {
+            systemTestFilePath = "src/test/resources/system_test.properties";
+        }
+        InputStream systemTestFileInputStream = new FileInputStream(new File(systemTestFilePath));
+        Properties systemTestProperties = new Properties();
+        systemTestProperties.load(systemTestFileInputStream);
+        for (Entry entry : systemTestProperties.entrySet()) {
+            String key = entry.getKey()
+                              .toString();
+            String valueInFile = entry.getValue()
+                                      .toString();
+            String value = getPeriorityValue(key, valueInFile);
+            System.setProperty(key, value);
+        }
+    }
+
+    public static String getPeriorityValue(String key, String value) {
```

Comment:
What is Periority? I'm assuming it should be getPropertyValue?

Reply:
The naming of the function aside, I'm missing javadoc. Especieally since the if statement are not crystal clear on first glance. Perhaps aoivd the shorthand form  and instead write out the if else statement to make the purpose of this code more readable.

Reply:
Javadoc added now

Reply:
method name is set as getPriorityValue()..
Method is simplified